storage/src/main/java/org/apache/uniffle/storage/request/CreateShuffleReadHandlerRequest.java

```diff
@@ -175,4 +176,12 @@ public Roaring64NavigableMap getExpectTaskIds() {
   public void setExpectTaskIds(Roaring64NavigableMap expectTaskIds) {
     this.expectTaskIds = expectTaskIds;
   }
+
+  public boolean isExpectedTaskIdsBitmapFilter() {
+    return expectedTaskIdsBitmapFilter;
+  }
+
+  public void setExpectedTaskIdsBitmapFilter(boolean expectedTaskIdsBitmapFilter) {
```

Comment:
`setExpectedTaskIdsBitmapFilter` -> `useExpectedTaskIdsBitmapFilter`?

Reply:
Done

Reply:
`useExpectedTaskIdsBitmapFilter()`,
The default value of expectedTaskIdsBitmapFilter is false. If we want to set the value , we can call the method `useExpectedTaskIdsBitmapFilter()`.